### DISCUSSION TITLE
Upgraded from v3.0.17 to v3.0.24, Safari iOS doesn't load stylesheet, no request in network panel

### DISCUSSION THREAD

**Discussion by alextes** [tailwindlabs/tailwindcss#8549] [Help] [answered]
**What version of Tailwind CSS are you using?**

3.0.17 / 3.0.24

**What build tool (or framework if it abstracts the build tool) are you using?**
```
"next": "^12.0.9",
```

For example: postcss-cli 8.3.1, Next.js 10.0.9, webpack 5.28.0

**What version of Node.js are you using?**
v18

**What browser are you using?**
Safari

**What operating system are you using?**
iOS

**Reproduction URL**

--

**Describe your issue**

Somehow, the stylesheet that is clearly listed in the head, and loaded fine in every browser, does not even show up as a request in the network tab for Safari on iOS. I just spent hours figuring out what in the world was going on. I'm mostly reporting this so that others may be helped by it. 

Below is what the tags look like in the final production build. Dev build works fine. I'm a 100% sure that it is the tailwind stylesheet specifically that the browser isn't loaded. I'm not that sure it is the .17 vs. .24 version that makes all the difference. I've been trying a lot of things and am currently trying to narrow down what exactly the fix was.

```html
<link rel="preload" href="/_next/static/css/7e2c1b5ff9af6eb3.css" as="style"/>
<link rel="stylesheet" href="/_next/static/css/7e2c1b5ff9af6eb3.css" data-n-g=""/>
```

**Comment by alextes** [comment]
@RobinMalfait maybe I'm misunderstanding how you triage in this repo but from my understanding, this is not a discussion. I'm trying to "fix an issue". If the bug report isn't detailed or reproducible enough for maintainers to want to do something with it, that's fine, please close the issue. That way it may stay easy to discover for others. As a discussion, it gets buried, and I imagine people looking to debug issues won't go searching discussions.

**Comment by alextes** [comment]
Alright, deployed version 3.0.24 again, immediately back to broken. Deployed the 3.0.17 version again, works.

So this is now definitely an issue with Safari and Tailwind, perhaps through some cross-interaction with NextJS/Webpack. Either way, unless I misunderstand what you view as "issues", this appears to be a rather serious issue. If maintainers disagree, totally fine of course, if the issue is indeed serious I'm sure more people will show up soon enough.

Thanks for a great CSS framework. Please move this back to issues. Closed or open.

**Comment by RobinMalfait** [comment, marked as answer]
Hey! Thank you for your bug report!
Much appreciated! 🙏

I moved this to a discussions because a reproduction was not provided and because Tailwind is not in charge of loading the CSS file, we only generate the css. 

If you click the new issue button then we default to discussions: https://github.com/tailwindlabs/tailwindcss/issues/new/choose

> **Get help**
>
> If you can't get something to work the way you expect, open a question in our discussion forums.

Which I believe is the case for your current issue.

Trying to create a reproduction ourselves from the information in an issue drastically increases the amount of time it takes for us to provide a solution and if we have do that for every issue it’s impossible for us to keep up.

If this is still an issue for you, please open a new issue but include a proper reproduction — a [Tailwind Play](https://play.tailwindcss.com) if the issue can be reproduced there, or a GitHub repository where you already have everything all set up so we can clone it and immediately see the issue without configuring everything by hand.

Sorry, not trying to be dismissive and definitely want to fix the issue if there is one, but we have to be strict about this one or there's no way we could keep up with the number of questions we get and the quality of the library would suffer as a result. Hope that's understandable!

---

Even though the message above mentioned that creation a reproduction ourselves takes too much time, I did it anyway since it's after hours here already. I created a fresh Next.js project and added Tailwind CSS. I did a production build using `next build` and ran `next start`.

Here is an example screenshot from Safari on iPhone.

![IMG_4060](https://user-images.githubusercontent.com/1834413/172736452-f5f61a21-6b3d-4518-ae2e-5e00d8b5390c.PNG)

That said, if you could create a reproduction repo and share it, then we can see if there is an actual issue with Tailwind and if there is then I'm more than happy to move this back to an issue.

Hope you understand! ❤️

**Reply by alextes** [reply]
Thanks for the explanation. A reproduction being a minimum requirement I find very reasonable for a popular lib! Perhaps it could be helpful if that's put in the template? Or maybe I even read over that, if so, sorry for that.

I'll try and create a reproduction. If I can't do it, let's close this and then others can show up and provide one if the issue hits more people 👍.

Thanks again for the great lib!

**Comment by alextes** [comment]
aaand, after trying for an hour, even the reproduction I mention I managed above, by switching between v3.0.17 and v3.0.24 and deploying to our production cluster, is now working fine. Frustrating, but also a waste of time at this point. If it happens again I'll try again. For now, I'll close on this unsatisfactory note and get back to being productive 😅 .

Sorry for wasting your time!
